buildrpm/ocne.spec

```diff
@@ -37,6 +37,25 @@ mkdir -p $GOPATH/src/%{MOD_PATH}
 ln -s `pwd` $GOPATH/src/%{MOD_PATH}/ocne
 pushd $GOPATH/src/%{MOD_PATH}/ocne
 
+# Check if code changes require updates to go.mod and/or the vendor folder.
+# After each step cleanup the download from "go mod tidy" and "go mod vendor,
+# it causes cannot find module providing packages errors
+go mod tidy
+rm -rf $GOPATH/pkg
```

Comment:
won't this make the build alot slower?

Reply:
A rough compare of my build times with main seems to indicate this change could add 30 seconds to the build.  I am not sure how the hash would improve speed, the extra time for the build would be due to downloading dependencies on go mod tidy.

This change will not affect local builds with the Makefile.